Updated support priority information with additional context/ examples (#435)
astronomer/docs: cloud/stable/07_resources/02_support.md

When you submit a ticket to Astronomer's Support Portal, keep the following best practices in mind.

#### 1. Always indicate priority

In order for our team to serve yours most effectively, it's critical that we understand the impact of all reported issues. Generally speaking, here are the guidelines we operate under:

- **P1:** Mission critical systems are down, no workaround is immediately available

    Examples:

>     - Scheduler is not heartbeating, and a restart didn't fix the issue
>     - All Celery workers are offline
>     - Extended periods of 503 errors (without under-provisioned web server)
>     - Astronomer outage (for example, the Docker registry is down)

Reviewer: ```suggestion
    - The Scheduler is not heartbeating and a restart didn't fix the issue.
    - All Celery Workers are offline.
    - Extended periods of `503` errors that are not solved by allocating more resources to the Webserver.
    - Astronomer outage, such as downtime in the Astronomer Docker Registry.
```

Author: Updating some of these to be complete sentences, so that we can keep periods in.

Reviewer: ^Sweet, good call. Looks good.

> - **P2:** Some major functionality of Astronomer/Astronomer-owned Airflow is severely impaired, but you are still able to run essential DAGs.
> 
>     Examples:
> 
>     - Webserver is unavailable
>     - You are unable to redeploy, but running DAGs are still functioning
>     - Scheduler is not working, but fixable on restart
>     - Task logs are missing

Reviewer: ```suggestion
- **P2:** Some major functionality in Astronomer or your Airflow Deployment is severely impaired, but you are still able to run essential DAGs.

    Examples:

    - The Airflow Webserver is unavailable.
    - You are unable to deploy code to your Deployment, but existing DAGs and tasks are running as expected.
    - The Scheduler is not working but it can be fixed by restarting it.
    - Task logs are missing in the Airflow UI.
```

> - **P3:** Partial, non-critical loss of functionality of Astronomer/Astronomer-owned Airflow.
> 
>     Examples:
> 
>     - A bug in the Astronomer UI
>     - CLI usage is impaired (e.g. Docker Image incompatibilities)
>     - Some Airflow issues that have code-based solutions
>     - Log alerts in the Astronomer platform

Reviewer: ```suggestion
- **P3:** Partial, non-critical loss of functionality of Astronomer or your Airflow Deployment on Astronomer.

    Examples:

    - A bug in the Astronomer UI.
    - Astronomer CLI usage is impaired or you're seeing incompatibility errors between installed packages.
    - Some Airflow issues that have code-based solutions.
    - Log alerts on Astronomer.
```

- **P4:** Questions, issues with code inside specific DAGs, and issues with Airflow that are not primarily owned by Astronomer.

    Examples:

>     - You can't find your Workspace
>     - Package incompatibilities caused by specific use cases
>     - Questions about a best approach for some action in Airflow or Astronomer

Reviewer: ```suggestion
    - You can't find your Workspace.
    - Package incompatibilities caused by a specific, complex use case.
    - Questions about best practices for an action in Airflow or on Astronomer.
```
